.circleci/config.yml

```diff
@@ -284,6 +284,10 @@ orbs:
               name: Execute Docker image build and upload
               command: tools/circle-build-and-push-docker.sh
 
+filters: &all_tags
+  tags:
+    only: /.*/
```

Comment:
I suggest to have the same rule for tags as we have on travis. https://github.com/esl/MongooseIM/blob/4b674dab32d2d27d33cb0cdb033e169d9bed0013/.travis.yml#L77

Reply:
Makes all the sense, except that in Travis that's under `branches` so I didn't think of it as tags. These CIs are confusing xD